# Signal the gripper to use the velocity configuration q
caros_common/Q velocity
---
# Acknowledgement of the service request
bool success
